src/lib.rs

```diff
@@ -713,5 +717,39 @@ pub fn parse_path(path: &PathBuf, config: ParseConfig) -> anyhow::Result<ParseOu
         return Err(anyhow!("Some log entries did not have compile id"));
     }
 
+    if config.inductor_provenance {
```

Comment:
Instead of having this be a config to pass to tlparse, can we have it so this logic turns on if there's at least one `inductor_provenance_tracking_node_mappings`? If we already turned it on in our structured logging, there's no reason to also have to flip a config on tlparse side

Reply:
@jamesjwu  I added it because it's a new feature and I don't want to break the normal tlparse flow accidentally. I'm thinking we can remove this flag after it has been running smoothly for a while.